### PR TITLE
creation of logout button

### PR DESCRIPTION
**Pull request by emmanuelle-c**

### CODE REVIEW

**Review by NicosMQ — APPROVED** [review]

File: client/src/components/Header/Header.jsx [hunk 1]
```diff
@@ -7,13 +7,17 @@ import logo from "../../assets/images/logos/logo-fond-0.png";
 import { useStatus } from "../../contexts/auth";
 
 function BurgerMenu() {
-  const { login, currentUser } = useStatus();
+  const { login, currentUser, setLogin } = useStatus();
   const [showLinks, setShowLinks] = useState(false);
+  const [showlog, Setshowlog] = useState(false);
```
Comment (VinchC) [inline client/src/components/Header/Header.jsx]: setShowlog (camelCase)

File: client/src/components/Header/Header.jsx [hunk 2]
```diff
@@ -7,13 +7,17 @@ import logo from "../../assets/images/logos/logo-fond-0.png";
...
 
   const handleShowLinks = () => {
     setShowLinks(!showLinks);
   };
 
+  const handleShowlog = () => {
+    Setshowlog(true);
```
Comment (VinchC) [inline client/src/components/Header/Header.jsx]: idem

File: client/src/pages/FirstPage/FirstPage.jsx [hunk 3]
```diff
@@ -1,20 +1,51 @@
...
+import { useStatus } from "../../contexts/auth";
 import ban from "../../assets/images/logos/banner.svg";
 import logo from "../../assets/images/logos/logo-fond-0.png";
 
-function FirstPage() {  
-  
+
+function FirstPage() {
+  const { login, currentUser, setLogin } = useStatus();
+  const [showlog, Setshowlog] = useState(false);
```
Comment (VinchC) [inline client/src/pages/FirstPage/FirstPage.jsx]: and setShowlog (camelCase)

File: client/src/pages/FirstPage/FirstPage.jsx [hunk 4]
```diff
@@ -1,20 +1,51 @@
...
+
+  const handleShowlog = () => {
+    Setshowlog(true);
```
Comment (VinchC) [inline client/src/pages/FirstPage/FirstPage.jsx]: idem

**Review by VinchC — CHANGES_REQUESTED** [review]

File: client/src/components/Header/Header.jsx (hunk 1 above)
Comment (emmanuelle-c) [reply client/src/components/Header/Header.jsx]: modified !

File: client/src/components/Header/Header.jsx (hunk 2 above)
Comment (emmanuelle-c) [reply client/src/components/Header/Header.jsx]: modified !

File: client/src/pages/FirstPage/FirstPage.jsx (hunk 3 above)
Comment (emmanuelle-c) [reply client/src/pages/FirstPage/FirstPage.jsx]: done

File: client/src/pages/FirstPage/FirstPage.jsx (hunk 4 above)
Comment (emmanuelle-c) [reply client/src/pages/FirstPage/FirstPage.jsx]: done

**Review by VinchC — APPROVED** [review]

**Pull request merged**